CadRevealFbxProvider/Attributes/ScaffoldingAttributeParser.cs

```diff
@@ -70,15 +72,53 @@ ScaffoldingMetadata scaffoldingMetadata
                 v =>
                 {
                     var kvp = new Dictionary<string, string>();
+
+                    // in some cases, description and weight can appear in several columns (different manufacturers of item parts)
+                    // these columns need to be merged into one
+
+                    var description = v
+                        .Headers.Select((h, i) => new { header = h, index = i })
+                        .Where(el => el.header.ToLower().Contains("description"))
+                        .Select(el =>
+                        {
+                            var manufacturerName = el.header.ToLower().Replace("description", String.Empty).ToUpper();
+                            var partDescription = v.Values[el.index];
+                            if (partDescription.Length > 0)
+                                return manufacturerName + partDescription;
```

Comment:
Should this have a separator? currently i guess this will be `aluhakledger beam` or similar ?

Reply:
The column name for haki parts is "HAKI Description", so after removing "Description", you will have "HAKI " left. Maybe I can write it into the comment. 
Aluhak columns have only "Description" in the column name, so you will be left with empty string.

Reply:
I would not depend on the spacing like that. I'd consider trimming the manufacturername and then add padding when its used like above.

Reply:
Fixed!